src/stores/AppStore.ts

```diff
@@ -2,6 +2,7 @@ import * as _ from "lodash";
 import {action, autorun, computed, flow, makeObservable, observable, ObservableMap, when} from "mobx";
 import * as Long from "long";
 import axios from "axios";
+import * as Semver from "semver";
```

Comment:
shouldn't there be an additional `package.json` change too?

Reply:
@veggiesaurus added `semver` to `package.json`. also updated `package-lock.json`